themes/twitter/twitter.scss

```diff
@@ -0,0 +1,177 @@
+    line-height: $header-height; 
+    font-size: 16px;
+    font-weight: 400;
+    color: #8899a6;
+    margin: 0;
+    color: $text-color;
+    border-bottom: 1px solid #e1e8ed;
+  }
+
+  p {
+    display: block;
+    text-align: center;
+    color: #66757f;
+    font-weight: 400;
+    font-size: 13px;
+    margin-top: 7px;
+  }
+
+  .sa-button-container {
+    background-color: #f5f8fa;
+    border-top: 1px solid #e1e8ed;
+    box-shadow: 0px -1px 0px white;
+    margin: -$padding;
+    margin-top: 0;
+  }
+  &[data-has-confirm-button=false][data-has-cancel-button=false] {
+    padding-bottom: 10px;
+    .sa-button-container {
+      display: none;
+    }
+  }
+
+  button {
+    border-radius: 2px;
+    box-shadow: none !important;
+    text-shadow: 0px -1px 0px rgba(black, 0.3);
+    margin: 17px 0px;
+    border-radius: 4px;
+    font-size: 14px;
+    font-weight: 600;
+    padding: 8px 16px;
+    position: relative;
+    &:focus, &.cancel:focus {
+      box-shadow: none !important;
+      &::before {
+        content: "";
+        position: absolute;
+        left: -5px;
+        top: -5px;
+        right: -5px;
+        bottom: -5px;
+        border: 2px solid #a5b0b4;
+        border-radius: 8px;
+      }
+    }
+
+    &.confirm {
+      background-color: #55acee !important;
+      background-image: linear-gradient(rgba(0,0,0,0),rgba(0,0,0,0.05));
+      -ms-filter: "progid:DXImageTransform.Microsoft.gradient(startColorstr=#00000000, endColorstr=#0C000000)";
+      border: 1px solid #3b88c3;
+      box-shadow: inset 0 1px 0 rgba(255,255,255,0.15);
+      margin-right: $padding;
+      &:hover {
+        background-color: #55acee;
+        background-image: linear-gradient(rgba(0,0,0,0),rgba(0,0,0,0.15));
+        -ms-filter: "progid:DXImageTransform.Microsoft.gradient(startColorstr=#00000000, endColorstr=#26000000)";
+        border-color: #3b88c3;
+      }
+    }
+    &.cancel {
+      color: #66757e;
+      background-color: #f5f8fa;
+      background-image: linear-gradient(#fff,#f5f8fa);
+      text-shadow: 0px -1px 0px white;
+      margin-right: 9px;
+      border: 1px solid #e1e8ed;
+      &:hover, &:focus:hover {
+        background-color: #e1e8ed;
+        background-image: linear-gradient(#fff,#e1e8ed);
+        -ms-filter: "progid:DXImageTransform.Microsoft.gradient(enabled=false)";
+        border-color: #e1e8ed;
+      }
+      &:focus {
+        background: #fff;
+        border-color: #fff;
+      }
+    }
+  }
+
+  .sa-icon:not(.sa-custom) {
+    transform: scale(0.72);
+    margin-bottom: -2px;
+    margin-top: -10px;
+  }
+
+  input {
+    border: 1px solid #e1e8ed;
+    border-radius: 3px;
+    padding: 10px 7px;
+    height: auto;
+    box-shadow: none;
+    font-size: 13px;
+    margin: 10px 0;
+    &:focus {
+      border-color: #94A1A6;
+      box-shadow: inset 0 0 0 1px rgba(77, 99, 107, 0.7);
+    }
+  }
+
+  fieldset .sa-input-error {
+    display: none;
+  }
+
+  .sa-error-container {
+    text-align: center;
```

Comment:
Properties should be ordered background-color, border, margin-bottom, text-align